### PR TITLE
fix: rename oa

### PR DESCRIPTION
**Pull request by simboonlong**

### CODE REVIEW

File: test/e2e/email.integration.test.js [hunk 1]
```diff
@@ -86,8 +86,8 @@ describe("email", () => {
         .set("Content-Type", "application/json")
         .set("Accept", "application/json")
         .send({
-          to: "example@opencerts.io",
-          data: ropstenCertificate
+          to: "example@tradetrust.io",
```
Comment (isaackps) [inline test/e2e/email.integration.test.js:89]: is this a valid email address?

File: src/storage/webpack.config.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
   target: "node",
   plugins: [
     new webpack.ProvidePlugin({
-      fetch: path.resolve(path.join(__dirname, ".", "fetch-shim"))
+      fetch: path.resolve(path.join(__dirname, "../", "fetch-shim"))
```
Comment (isaackps) [inline src/storage/webpack.config.js:14]: y do we need this "../"? is it not working previously with the "."?

File: test/e2e/email.integration.test.js (hunk 1 above)
Comment (simboonlong) [reply test/e2e/email.integration.test.js:89]: it's `example@`, i'm assuming it's a dummy address and not supposed to use anyway? 😬

File: src/storage/webpack.config.js (hunk 2 above)
Comment (simboonlong) [reply src/storage/webpack.config.js:14]: have moved `fetch-shim.js` up 1 level

File: test/e2e/email.integration.test.js (hunk 1 above)
Comment (isaackps) [reply test/e2e/email.integration.test.js:89]: hahaha ok

File: src/storage/webpack.config.js (hunk 2 above)
Comment (isaackps) [reply src/storage/webpack.config.js:14]: ok

**Review by isaackps — APPROVED** [review]

**Pull request merged**